src/ldap-xplat.c

```diff
@@ -722,15 +722,13 @@ ldap_init_thread_func(void *params) {
         set_cert_policy(data->ld, data->cert_policy);
     }
 
-#if !defined(WIN32) && !defined(MACOSX) && LDAP_VENDOR_VERSION > 20443
```

Comment:
This check still necessary for Windows, because WinLDAP doesn't know these settings. I also lean to keep the version check and only remove ruling out Mac users.